docs/_includes/usage/cf-core/usage.md

```diff
@@ -1134,6 +1134,10 @@ Links within a `nav` element are not underlined.
 </table>
 ```
 
+_Note: While `th` elements normally only contain raw text, they may sometimes
```

Comment:
In a perfect world we'd be consistent with our notes in CF, but this is fine till design manual party time.